payjoin/src/receive/error.rs

```diff
@@ -260,6 +260,8 @@ pub(crate) enum InternalSelectionError {
     TooManyOutputs,
     /// No selection candidates improve privacy
     NotFound,
+    /// Missing previous txout information
+    PrevTxOut(crate::psbt::PrevTxOutError),
```

Comment:
This smells to me because `MissingPrevTxOut` does not seem like a selection error